app.rb

```diff
@@ -113,4 +113,25 @@ def list_all_rentals
   def exit
     puts 'Thank you for using the app, see you later!'
   end
+
+  def select_option(user_choice)
+    options = {
+      '1' => :list_all_books,
+      '2' => :list_all_people,
+      '3' => :create_person,
+      '4' => :create_book,
+      '5' => :create_rental,
+      '6' => :list_all_rentals,
+      '7' => :exit
+    }
+
+    method = options[user_choice]
+    if method.nil?
+      puts 'Invalid option, please try again!'
+      puts
+      show_menu
+    else
+      send(method)
+    end
+  end
```

Comment:
- [ ] All options and input requests should be stored in the main.rb, Please move this part back into main.rb. All methods for options should be stored in the app class as you have done. But please ensure your methods in the app class don't have any line for asking users for an option as I mentioned above.